### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/project-generation.md`

### PR DESCRIPTION
**Pull request by BilalAlpaslan**
Turkish translate for project-generation.md
Main issue: https://github.com/tiangolo/fastapi/issues/1892

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 0d3fa7e7db0fef56b647175a4b0fcde36c1a20a1 at: https://649a251ab8e9671a3f1c206a--fastapi.netlify.app

File: docs/tr/docs/project-generation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* Docker Swarm Mode ile deployment.
+* **Docker Compose** entegrasyonu ve lokal geliştirme için optimizasyon.
+* Uvicorn ve Gunicorn ile **Production ready** Python web server'ı.
+* Python <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">**FastAPI**</a> backend:
+    * **Hızlı**: **NodeJS** ve **Go** ile eşit, çok yüksek performans (Starlette ve Pydantic'e teşekkürler).
+    * **Sezgisel**: Editor desteğı. <abbr title="auto-complete, IntelliSense gibi isimlerle de bilinir">Otomatik tamamlama</abbr>. Daha az debugging.
+    * **Kolay**: Kolay öğrenip kolay kullanmak için tasarlandı. Daha az döküman okuma daha çok iş.
+    * **Kısa**: Minimum kod tekrarı. Her parametre bildiriminde birden çok özellik.
+    * **Güçlü**: Production-ready. Otomatik interaktif dökümantasyon.
+    * **Standartlara dayalı**: API'ler için açık standartlara dayanır (ve tamammen uyumludur): <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> ve <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Şeması</a>.
```
Comment (cun3yt) [inline docs/tr/docs/project-generation.md:23]: `tamammen` => `tamamen`

File: docs/tr/docs/project-generation.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    * <a href="https://fastapi.tiangolo.com/features/" class="external-link" target="_blank">**Birçok diger özelliği**</a> dahili otomatik doğrulama, serialization, interaktif dokümantasyon, OAuth2 JWT token ile authentication, vb.
+* **Güvenli şifreleme** .
+* **JWT token** authentication.
```
Comment (cun3yt) [inline docs/tr/docs/project-generation.md:26]: authentication => kimlik doğrulama ?

File: docs/tr/docs/project-generation.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* **SQLAlchemy** models (Flask dan bağımsızdır. Celery worker'ları ile kullanılabilir).
+* Kullanıcılar için temel başlangıç ​​modeli (gerektiği gibi değiştirin ve kaldırın).
+* **Alembic** migration.
+* **CORS** (Cross Origin Resource Sharing).
+* **Celery** worker'ları ile backend içerisinden seçilen işleri çalıştırabilirsiniz.
+* **Pytest**'e dayalı, Docker ile entegre REST backend testleri ıle veritabanından bağımsız olarak tam API etkileşimini test edebilirsiniz. Docker'da çalıştığı için her seferinde sıfırdan yeni bir veri deposu oluşturabilir (böylece ElasticSearch, MongoDB, CouchDB veya ne istersen kullanabilirsin ve sadece API'nin çalışıp çalışmadığını test edebilirsin).
```
Comment (cun3yt) [inline docs/tr/docs/project-generation.md:32]: `ıle` => `ile`

File: docs/tr/docs/project-generation.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* Let's Encrypt **HTTPS** sertifikalarının otomatik oluşturulması dahil olmak üzere Traefik entegrasyonu.
+* GitLab **CI** (sürekli entegrasyon), backend ve frontend testi dahil.
+
+## Full Stack FastAPI Couchbase
+
+GitHub: <a href="https://github.com/tiangolo/full-stack-fastapi-couchbase" class="external-link" target="_blank">https://github.com/tiangolo/full-stack-fastapi-couchbase</a>
+
+⚠️ **UYARI** ⚠️
+
+sıfırdan bir projeye başlıyorsanız alternatiflerine bakın.
```
Comment (cun3yt) [inline docs/tr/docs/project-generation.md:61]: `sıfırdan` => `Sıfırdan`

**Review by cun3yt — CHANGES_REQUESTED** [review]
A few small change requests. Happy to approve after they are resolved.

**Comment by tiangolo** [comment]
📝 Docs preview for commit a0ba04c90c02648a82d9eb15f8b1e5ba5a005210 at: https://37f80710.fastapitiangolo.pages.dev

File: docs/tr/docs/project-generation.md (hunk 1 above)
Comment (cun3yt) [reply docs/tr/docs/project-generation.md:23]: Bir de bu vardi ^^

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7439779b7d695a9d4571b8b1c2e5fade6411d36c at: https://3d0e8e7c.fastapitiangolo.pages.dev

**Review by cun3yt — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for you contribution @BilalAlpaslan 😸 

And thanks for the review @cun3yt 💯

**Pull request merged**